sysdig/internal/client/v2/posture_policies.go

```diff
@@ -27,7 +31,46 @@ func (client *Client) ListPosturePolicies(ctx context.Context) ([]PosturePolicy,
 
 	return resp.Data, nil
 }
+func (client *Client) CreateOrUpdatePosturePolicy(ctx context.Context, p *CreatePosturePolicy) (*PosturePolicy, string, error) {
+	payload, err := Marshal(p)
+	if err != nil {
+		return nil, "", err
+	}
+	response, err := client.requester.Request(ctx, http.MethodPost, client.getPosturePolicyURL(PosturePolicyCreatePath), payload)
+	if err != nil {
+		return nil, "", err
+	}
+	defer response.Body.Close()
+	if response.StatusCode != http.StatusOK && response.StatusCode != http.StatusCreated && response.StatusCode != http.StatusAccepted {
```

Comment:
why checking all those status codes? Do you know what is the status code of the service?

Reply:
201 for create
200 for update
accept status code removed